Lecture-3.md

```diff
@@ -1403,7 +1403,7 @@ console.log( A(3)(4)(5) );
 - Objects share functions.
 - All members are public.
 - An object constructor looks like a normal function.
-- Prexing a call to a function with the keyword `new` indicates to the JavaScript runtime that the function should behave like a constructor.
+- Prefexing a call to a function with the keyword `new` indicates to the JavaScript runtime that the function should behave like a constructor.
```

Comment:
Prefixing? 🙂

Reply:
Oh I am sorry, I thought it was a typo. Can I remove this change from the pull request or I have to delete it? It's only my second pull request and I am still learning how they work.

Reply:
@lyrakisk it is, but I believe your correction contains a new typo. The correct spelling is `Prefixing` rather than `Prefexing` right? 😃

Reply:
As for your Pull Request; you can simply push a new change on top of the current one. 
The Pull Request cycle is as follows;

1. Propose changes
2. Get feedback
3. Make new changes following from the feedback (if any; then also go back to 2).
4. Get your changes merged.